src/modules/dumb/dumb.ino

```diff
@@ -1,21 +1,40 @@
 #include <Wire.h>
 
-#define BUTTON (4)
-#define VIN_RED_LED (2)
-#define VIN_GREEN_LED (3)
-
-// Addresses pins
-int as[] = {8, 9, 10, 11};
+int as[] = {8, 9, 10, 11}; // Pins used to read the module's address
 
 uint16_t timeleft = 0xFFFF;
 uint16_t difficulty = 1;
 
+
+bool masterNeedsDefusingInformation = false;
 uint16_t defused = 0;
+uint16_t penality = 0;
+
+uint16_t command = 0;   // Last command sent by master
+uint16_t parameter = 0; // Last command's parameter sent by master
+
+// Output
+#define RED_LED (2)
+#define GREEN_LED (3)
+#define SENSE_PIN (7)
+
+// Input
+#define BUTTON (4)
+
+// Command
```

Comment:
This defines commands and such, `_CMD` part must prefix the command type (`CMD_TIME`)

Reply:
Pourquoi ?